snap/snapcraft.yaml

```
@@ -1,52 +1,128 @@
-name: spark-k8s # you probably want to 'snapcraft register <name>'
+name: spark-client
 base: core20
-version: '0.1'
-summary: Client side snap to submit spark jobs to kubernetes.
+version: '3.3.0'
+summary: Client side scripts to submit Spark jobs to a cluster.
 description: |
-  Spark is now available on Kubernetes and needs a client to submit and 
-  interact with jobs. This snap packages the client binaries for the same.
+  The spark-client snap includes the scripts spark-submit, spark-shell, pyspark and other tools for managing Apache Spark jobs.
 
-grade: devel
+grade: stable
 confinement: strict
 
+plugs:
+  enable-scala-history:
```

By convention, when you ask on forum.snapcraft.io to have this autoconnected (which will mean you don't need to call out manually connecting in the README), they'll probably ask that you call this something like `dot-scala-history`

```suggestion
  dot-scala-history:
```